src/densenet.jl

```diff
@@ -56,22 +56,22 @@ function densenet_layers()
     weights[string(ele)] = convert(Array{Float64, N} where N ,weight[ele])
   end
   ls = _densenet()
-  ls[1].weight.data .= weights["conv1_w_0"]
+  ls[1].weight.data .= weights["conv1_w_0"][end:-1:1,:,:,:][:,end:-1:1,:,:]
   ls[2].β.data .= weights["conv1/bn_b_0"]
   ls[2].γ.data .= weights["conv1/bn_w_0"]
   l = 4
   for (c, n) in enumerate([6, 12, 24, 16])
       for i in 1:n
           for j in [2, 4]
-              ls[l][i].layer[j].weight.data .= weights["conv$(c+1)_$i/x$(j÷2)_w_0"]
+              ls[l][i].layer[j].weight.data .= weights["conv$(c+1)_$i/x$(j÷2)_w_0"][end:-1:1,:,:,:][:,end:-1:1,:,:]
               ls[l][i].layer[j-1].β.data .= weights["conv$(c+1)_$i/x$(j÷2)/bn_w_0"]
               ls[l][i].layer[j-1].γ.data .= weights["conv$(c+1)_$i/x$(j÷2)/bn_w_0"]
```

Comment:
It looks to me like we're taking the same weight (`weights["conv$(c+1)_$i/x$(j÷2)/bn_w_0"]`) and assigning it to both beta and gamma here.  This looks like a copy-paste error.

Reply:
Yes its a copy-paste error. Fixing it